README.md

```diff
@@ -187,9 +188,6 @@ If you want to test it locally, see [Docker](#docker).
 | `ui.header`                                     | Header at the top of the dashboard.                                                                                                                | `Health Status`            |
 | `ui.logo`                                       | URL to the logo to display.                                                                                                                        | `""`                       |
 | `ui.link`                                       | Link to open when the logo is clicked.                                                                                                             | `""`                       |
-| `ui.buttons`                                    | List of buttons to display below the header.                                                                                                       | `[]`                       |
-| `ui.buttons[].name`                             | Text to display on the button.                                                                                                                     | Required `""`              |
-| `ui.buttons[].link`                             | Link to open when the button is clicked.                                                                                                           | Required `""`              |
```

Comment:
Why is this being removed? the root-level `ui` configuration is different from the endpoint-level `ui` configuration.

Also, I'm sorry for making you wait for so long - thank you for your patience :)

Reply:
idk, wasn't aware of this removal at all. We rushed it via the online diff editor. Maybe this is sourced in another branch/merge and my branch isn't on yet. This feature is completly new to me.
Will readd it asap